Dockerfile.proto

```diff
@@ -0,0 +1,66 @@
+
+ARG UBUNTU_VERSION=22.04
+ARG TFSERVING_VERSION=2.9.0
+
+# Build image, genearates proto files
+FROM arm64v8/ubuntu:${UBUNTU_VERSION}  AS build-image
+
+ARG TFSERVING_VERSION
+
+RUN <<EOF
+apt-get update && apt-get install -y --no-install-recommends \
+python3 \
+python3-pip \
+git \
+&& rm -rf /var/lib/apt/lists/*
+EOF
+
+# grpcio-tools contains protoc, which allows to build the pb2.py files in the install-tf.sh script.
+# Installing directly grpcio-tools result in an older version of protobuf.
+# So we install first the dependencies with the desired version and then grpcio-tool without dependencies.
+RUN pip install protobuf==4.21.1 six==1.16.0 grpcio==1.46.3 --no-cache-dir
+RUN pip install --no-dependencies grpcio-tools==1.47.0
+
+#Build tensorflow serving
+WORKDIR /build/tf
```

Comment:
Just a suggestion to group build files under a proto file.
```suggestion
# Build TensorFlow serving proto files
WORKDIR /build/proto/tf
```

Reply:
The relevant file will already end up in
build/(tf|vdo|param)/proto_utils
which is a "clean directory" if that is the reason

Reply:
Ack